test/gui/shared/steps/steps.py

```diff
@@ -1466,3 +1463,38 @@ def step(context, username, foldername):
 @Then("credentials wizard should be visible")
 def step(context):
     waitForObject(AccountConnectionWizard.CREDENTIAL_PAGE)
+
+
+@When("user selects configure_synchronization_manually option in advanced section")
+def step(context):
+    clickButton(
+        waitForObject(
+            names.syncModeGroupBox_configureSyncManuallyRadioButton_QRadioButton
+        )
+    )
+    clickButton(waitForObject(names.setupWizardWindow_nextButton_QPushButton))
+
+
+@Then("dialog add folder sync connection should be visible")
+def step(context):
+    clickButton(
+        waitForObject(
+            names.add_Folder_Sync_Connection_qt_passive_wizardbutton1_QPushButton
+        )
+    )
+
```

Comment:
Do not perform action in `Then` step. If you need to do this action then make it a `When` step instead.